graph/src/blockchain/mod.rs

```diff
@@ -36,10 +36,7 @@ use std::{collections::BTreeMap, fmt::Debug};
 use std::{collections::HashMap, convert::TryFrom};
 use web3::types::H256;
 
-pub use block_stream::{
-    BlockStream, BlockStreamMetrics, ChainHeadUpdateListener, ChainHeadUpdateStream,
-    TriggersAdapter,
-};
+pub use block_stream::{BlockStream, BlockStreamMetrics, ChainHeadUpdateListener, TriggersAdapter};
```

Comment:
Do I have some ide/rustfmt misconfiguration?

Reply:
This wasn't just a reformat, the `ChainHeadUpdateStream` import was removed. And rustfmt runs on CI so you never have to worry about that.